pkg/authorization/authorizer/remote/authorizer.go

```diff
@@ -93,21 +93,19 @@ func (r *RemoteAuthorizer) GetAllowedSubjects(ctx kapi.Context, attributes autho
 }
 
 func getAction(namespace string, attributes authorizer.Action) authzapi.Action {
+	resource := attributes.GetResource()
```

Comment:
Do we have tests for round tripping `authorizer.Action` `<->` `authzapi.Action`.  Maybe we don't need them if we completely use kube stuff...